zebra-utils/zcash-rpc-diff

```diff
@@ -192,6 +195,37 @@ if [ "$1" == "getaddressbalance" ]; then
     fi
 fi
 
+if [ "$1" == "getrawmempool" ] && [ $CHECK_EXIT_STATUS != 0 ]; then
+    echo
+    echo "Obtaining transaction details"
+    echo
+
+    set TRANSACTION_ID
+    set TRANSACTION_HEX
+    set TRANSACTION_DECODED
+
+    while IFS= read -r line
+    do
+        if [ "$line" != "[" ] && [ "$line" != "]" ]; then
```

Comment:
Optional: This parsing works, but it is fragile - any output format changes will break it, including extra spaces or new lines.

If we use jq, then it will always parse the JSON correctly. It would be something like:
```sh
ZEBRAD_TRANSACTION_IDS=$(cat "$ZEBRAD_RESPONSE" | $JQ -r 'join(" ")')
for TRANSACTION_ID in $ZEBRAD_TRANSACTION_IDS; do
    …
done
```

Reply:
This is better, used in https://github.com/ZcashFoundation/zebra/pull/6035/commits/dd1d35c81169335fe5fd496239f453118ac88034

Thanks.